tee-worker/local-setup/launch.py

```diff
@@ -156,10 +157,16 @@ def offset_port(offset):
 
 
 def setup_environment(offset, config, parachain_dir):
+    if not os.path.isfile("./local-setup/.env"):
+        shutil.copy("./local-setup/.env.dev", "./local-setup/.env")
+
```

Comment:
Is this needed?
I see the following line is 
```
load_dotenv("./local-setup/.env.dev")
```

Reply:
thanks. fixed.